crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -220,6 +272,17 @@ pub(crate) trait StateReadExt: StateRead {
         Ok(fee)
     }
 
+    #[instrument(skip(self))]
+    async fn get_bridge_sudo_change_fee(&self) -> Result<u128> {
```

Comment:
This behaivior means you just won't be able to do the action unless fee is set, so if we get fee action updated you can't use these features until we set the fee right? That's a kinda nifty way to activate new actions.

Reply:
yess that is true!